user_guide_src/source/outgoing/localization/008.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
-'languageKey' => [
-    'nested' => [
-        'key' => 'The actual message to be shown.',
+[
+    'languageKey' => [
+        'nested' => [
+            'key' => 'The actual message to be shown.',
+        ],
     ],
-],
+];
```

Comment:
What about this one and 009.php?
From the RST I can not really judge what they are supposed to be.

Reply:
They should have a return directive.